src/pages/mesh/release/index.md

```diff
@@ -18,6 +18,18 @@ import UpdateNotice from '/src/_includes/update-notice.md'
 
 The following sections list updates to API Mesh for Adobe Developer App Builder. Refer to the [Upgrade version](upgrade.md) for more information on upgrading.
 
+## February 19, 2025
+
+This release contains the following changes to API Mesh:
+
+### Enhancements
```

Comment:
Can you also add a note about https://github.com/adobe/aio-cli-plugin-api-mesh/pull/206 ?

Reply:
@revanth0212 done